src/build-tools/near-bindgen-exporter.js

```diff
@@ -67,3 +71,19 @@ export default function () {
     }
   };
 }
+
+
+function getCFunctions(){
+  const builderRaw = readFileSync(path.resolve('../cli/builder/builder.c'), 'utf-8')
+  const regex = new RegExp(/(?<=JS_SetPropertyStr\(ctx, env, ").+?(?=", JS_NewCFunction\(ctx,)/gm)
```

Comment:
There are more functions to filter out, basically every C function that's included in builder.c, such as `strncpy`, so this is quite challenging to figure out all C function names. I wonder whether one these directions can be easier:
- when confliction happens, can we collect the C compiler's error/warning and indicate the user to rename the contract method
- can we iterate contract method names like you did, then for any name confliction, use some C code-transformation / compiler tool to rename C functions to some random names. For example, a developer can write a contract method named `strncpy` (:joy:), and we let the tool to rename original `strncpy` to `___strncpy`

Reply:
Wooow, yes. Why not, I will update PR